autoload/replant/handle.vim

```diff
@@ -234,9 +230,12 @@ fun! replant#handle#apropos_fzf(msgs)
 
   let Resolver = {a -> get(actions, a, ['jump', 'edit'])}
 
-  let opts = {'source': resources,
+  let file = tempname()
+  let preview_command = 'grep -ae {}: '.file.' | cut -d : -f 2-9999 | xargs --null printf "%s\n" | sed -e "s/^[[:space:]]*//"'
```

Comment:
I think you're right to worry about portability, I've got something in my head about a particular `cut` feature being different under BSD cut, but that may be in my head. Grep & regex can be pretty perilous here too.

I worry about the regex symbols that could be in `{}` like `*` and `.` throwing out the regex approach. I suppose technically there can also be symbols like `:'.

I did come up with this alternative fzf command, which isn't perfect, but is quite good:

```
fzf -d ':' --preview 'echo {2..}' --with-nth=1
```

I created a file like:

```
foo: blah blah
a.x/core!: yjfodf: sdkfksdjf
```

The `with-nth` allows restricting display to just the symbol, the only drawback being that it displays the delimiter, so the lines look like:

```
a.x/core!:
```

Ref: https://github.com/junegunn/fzf/pull/1253

Reply:
Will take a little look and see how it can be improved.  I like the fzf option you described, will be interested to see how well that works.